eng/WpfArcadeSdk/tools/ShippingProjects.props

```diff
@@ -52,24 +52,55 @@
     <WindowsDesktopSdkProject>
       PresentationBuildTasks
     </WindowsDesktopSdkProject>
-    
+
     <CycleBreakerProjects>
-        PresentationFramework-PresentationUI-api-cycle;
-        PresentationFramework-ReachFramework-impl-cycle;
-        PresentationFramework-System.Printing-api-cycle;
-        PresentationFramework-System.Printing-impl-cycle;
-        PresentationUI-PresentationFramework-impl-cycle;
-        ReachFramework-PresentationFramework-api-cycle;
-        ReachFramework-System.Printing-api-cycle;
-        System.Printing-PresentationFramework-api-cycle
+      PresentationFramework-PresentationUI-api-cycle;
+      PresentationFramework-ReachFramework-impl-cycle;
+      PresentationFramework-System.Printing-api-cycle;
+      PresentationFramework-System.Printing-impl-cycle;
+      PresentationUI-PresentationFramework-impl-cycle;
+      ReachFramework-PresentationFramework-api-cycle;
+      ReachFramework-System.Printing-api-cycle;
+      System.Printing-PresentationFramework-api-cycle
     </CycleBreakerProjects>
-    <HandCraftedReferenceProjects>
-        System.Printing-ref
-    </HandCraftedReferenceProjects>
+
+    <!-- 
+      Reference assembly projects are categorized by what repo they are shipping in.
```

Comment:
Can these all be put in external repo under shared, then consumed from Microsoft.DotNet.Arcade.Wpf.Sdk (which brings along all sources under Shared\**/* ?

Reply:
We'd still need logic to disable picking up ref outputs in the internal packaging, so I am not sure we'd reduce complexity by doing that.

Reply:
We could probably start moving most stuff out like that though, including cycle breakers.